src/native/utils/game-loop.js

```diff
@@ -22,7 +23,7 @@ export default class GameLoop {
   }
 
   unsubscribe(id) {
-    this.subscribers.splice((id - 1), 1);
+    delete this.subscribers[id - 1];
```

Comment:
This change will mean that for something like:

```js
subscribers = [1, 2, 3]
```

If removing `id = 2` we'd end up with:

```js
[1, 3]
````

but now would instead get:

```js
[1, empty, 3]
```

@kenwheeler -- Does the above change look ok?

Reply:
👍